pkgs/development/python-modules/manuf/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, runCommand
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "manuf";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "coolbho3k";
+    repo = "manuf";
+    rev = "${version}";
+    sha256 = "sha256-3CFs3aqwE8rZPwU1QBqAGxNHT5jg7ymG12yBD56gTNI=";
+  };
+
+  # Tests appeared to be broken
+  doCheck = false;
+
+  passthru.tests = {
+    testMacAddress = runCommand "${pname}-test" {} ''
+      ${python3.pkgs.manuf}/bin/manuf BC:EE:7B:00:00:00 > $out
+      [ "$(cat $out | tr -d '\n')" = "Vendor(manuf='ASUSTekC', manuf_long='ASUSTek COMPUTER INC.', comment=None)" ]
+    '';
+  };
+
```

Comment:
Could you please enable pythonImportsCheck for the simple check?

Reply:
I don't think that the main python file imports anything outside the standard library. Of course, I can make a check that standard library has all imports required if it is necessary.

Although tests have some required packages, but they are turned off.

Reply:
We can find that its dependencies are all standard libraries, but there are many packages that import implicit dependencies such as `pkg_resources` or `importlib_metadata`.
Adding this check would be a good choice since it has little overhead.

Reply:
Added `pythonImportsCheck`. But not sure if I did it correctly.